cardinal/chain/adapter.go

```diff
@@ -17,8 +18,18 @@ import (
 
 // Adapter is a type that helps facilitate communication with the EVM base shard.
 type Adapter interface {
```

Comment:
nit: can we call this ShardAdapter

Reply:
if we want to go that direction of naming, i would suggest to rather rename the `chain` package to `shard`

usage then looks like:
```go
import "github.com/argus-labs/world-engine/cardinal/shard"

func Foo() {
    a := shard.NewAdapter()
    ...
}
```

keeps things slimmed down and achieves the same goal

Reply:
sounds good